helm/templates/coder.yaml

```diff
@@ -30,9 +30,19 @@ spec:
       restartPolicy: Always
       {{- with .Values.coder.image.pullSecrets }}
       imagePullSecrets:
-        {{- toYaml . | nindent 8 }}
+      {{- toYaml . | nindent 8 }}
       {{- end }}
       terminationGracePeriodSeconds: 60
+      {{- if .Values.coder.affinity }}
+      affinity: {{- toYaml .Values.coder.affinity | nindent 8 }}
+      {{- end }}
```

Comment:
This should be a `with` like the other ones, and the `toYaml ...` command should be on it's own line like the others also.

Reply:
@deansheather I'm receiving this error as a result of doing this:

```
Error: template: coder/templates/coder.yaml:38:24: executing "coder/templates/coder.yaml" at <.Values.coder.affinity>: nil pointer evaluating interface {}.code
```